estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class TestModel(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'Estate Model Offer'
+
+    price = fields.Float()
+    status = fields.Selection(copy=False, selection=[('Accepted', 'Accepted'), ('Refused', 'Refused')])
```

Comment:
```suggestion
    status = fields.Selection(copy=False, selection=[('accepted', 'Accepted'), ('refused', 'Refused')])
```
Prefer using snake_case keys.
We basically try to fit [PEP8](https://peps.python.org/pep-0008/) standards